jsontableschema/types/datetime.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
```

Comment:
About constraints camelCasing and this:
Here I use rule like type/constraint from spec is mapping 1-to-1 to type/constraint file. So it could be useful while this lib is reference.